src/Command/Environment/EnvironmentActivateCommand.php

```diff
@@ -112,7 +112,14 @@ protected function activateMultiple(array $environments, InputInterface $input,
                 }
                 continue;
             }
+
+            $hasGuaranteedCPU = $this->api()->environmentHasGuaranteedCPU($environment);
             $question = "Are you sure you want to activate the environment " . $this->api()->getEnvironmentLabel($environment) . "?";
+            if ($resourcesInit === 'parent' && $hasGuaranteedCPU && $this->config()->has('warnings.guaranteed_resources_msg')) {
+                $question = trim($this->config()->get('warnings.guaranteed_resources_msg'))
+                    . "\n\n" . "Are you sure you want to activate the environment " . $this->api()->getEnvironmentLabel($environment) . "?";
```

Comment:
The full message looks something like:

https://github.com/platformsh/cli/blob/b1a907bfb6154fcdbb508390247634c782400bba/internal/config/upsun-cli.yaml#L92-L96

```
You have chosen to allocate guaranteed resources for app(s) and/or service(s).
This change may affect resource costs. See: https://upsun.com/pricing/

This process requires a redeployment of containers on their own host, which may take a few minutes to complete.

Are you sure you want to activate the environment Foo (foo) (type: development)?
```

I'm not sure this makes sense: the "redeployment", "allocate", and "change" all indicate an existing environment that is being altered, not a newly activated/branched environment.

Reply:
fixed!

Reply:
```suggestion
                    . "\n\n" . $question;
```